tailwind.config.js

```diff
@@ -11,5 +11,5 @@ module.exports = {
       },
     },
   },
-  plugins: [],
+  plugins: [require('@tailwindcss/aspect-ratio'),],
```

Comment:
Do you think we still need this plugin?

We are using Tailwind 3 which has official support for [aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) and the property has [pretty good support across the browsers](https://caniuse.com/?search=aspect-ratio).

If we choose to keep it, then we should [disable the core plugin](https://github.com/tailwindlabs/tailwindcss-aspect-ratio#installation) to avoid conflicts:

```js
// tailwind.config.js
module.exports = {
  theme: {
    // ...
  },
  corePlugins: {
    aspectRatio: false,
  },
  plugins: [
    require('@tailwindcss/aspect-ratio'),
    // ...
  ],
}
```

Reply:
Thanks for pointing this out. I still think the plugin is needed. I have tried to use the native CSS utilities for aspect ratio, but it does not produce the expected output. I will continue to follow it up.